### PR TITLE
create petition if petition view is included on page

### PR DESCRIPTION
**Pull request by tanyaparker**

### CODE REVIEW

File: pegasus/sites.v3/code.org/public/promote/index.haml [hunk 1]
```diff
@@ -64,9 +63,10 @@ social:
 
 :javascript
   var petition;
-  var promote;
```
Comment (islemaster) [inline pegasus/sites.v3/code.org/public/promote/index.haml:67]: Related?

**Comment by breville** [comment]
lgtm, assuming it's tested and validated no more browser errors.

File: pegasus/sites.v3/code.org/public/promote/index.haml (hunk 1 above)
Comment (tanyaparker) [inline pegasus/sites.v3/code.org/public/promote/index.haml:67]: Not related to this particular error. It was just some left over js that I cleaned up. Would it have been better to do it separately?

Comment (islemaster) [inline pegasus/sites.v3/code.org/public/promote/index.haml:67]: Fine by me, just curious.

**Pull request merged**